fluss-client/src/main/java/org/apache/fluss/client/table/scanner/log/ScanRecords.java

```diff
@@ -37,7 +37,9 @@
  */
 @PublicEvolving
 public class ScanRecords implements Iterable<ScanRecord> {
-    public static final ScanRecords EMPTY = new ScanRecords(Collections.emptyMap());
+    public static final ScanRecords empty() {
+        return new ScanRecords(Collections.emptyMap());
+    }
```

Comment:
Not necessary? I still prefer the preivous implementation because it can avoid some small object overhead (GC).